module/apmgorm/apmgorm_test.go

```diff
@@ -148,30 +151,63 @@ func TestWithContextNonSampled(t *testing.T) {
 }
 
 func TestCaptureErrors(t *testing.T) {
-	db, err := apmgorm.Open("sqlite3", ":memory:")
-	require.NoError(t, err)
-	defer db.Close()
+	t.Run("sqlite3", func(t *testing.T) {
+		db, err := apmgorm.Open("sqlite3", ":memory:")
+		require.NoError(t, err)
+		testCaptureErrors(t, db)
+		defer db.Close()
+	})
+
+	if os.Getenv("PGHOST") == "" {
+		t.Logf("PGHOST not specified, skipping")
+	} else {
+		t.Run("postgres", func(t *testing.T) {
+			db, err := apmgorm.Open("postgres", "user=postgres password=hunter2 dbname=test_db sslmode=disable")
+			require.NoError(t, err)
+			testCaptureErrors(t, db)
+			defer db.Close()
```

Comment:
`defer db.Close()` needs to be just above `testCaptureErrors(t, db)`, in case testCaptureErrors panics

Reply:
That's right. I've just fixed it.